gdal/ogr/ogrutils.cpp

```diff
@@ -1223,13 +1223,23 @@ int OGRParseXMLDateTime( const char* pszXMLDateTime,
              (c == '+' || c == '-') )
     {
         TZ = 100 + ((c == '+') ? 1 : -1) * ((TZHour * 60 + TZMinute) / 15);
+        if( second >= 59.99)
```

Comment:
When is this change really necessary ? There is an edge case where it is not correct: leap seconds where second = 60.x is valid